packages/web-frontend/src/historyNavigation/utils.js

```diff
@@ -42,19 +42,20 @@ export const createLocation = params => {
     if (value !== undefined) searchComponents[component] = value;
   });
 
-  return { pathname, search: searchComponents };
+  return { pathname, search: translateSearchToExternal(searchComponents) };
```

Comment:
Two notes about `translateSearchToInternal`, `translateSearchToExternal`:

* We do not have to export them now, since they are used only in this file 🎉 
* `internal` and `external` are not explanatory enough, we could rename those to
`translateSearchToInternal` => `decodeSearch`
`translateSearchToExternal` => `stringifySearch`

Reply:
Good point!